latch/registry/table.py

```diff
@@ -464,19 +549,27 @@ def commit(self) -> None:
         """
         updates: List[l.SelectionNode] = []
```

Comment:
aren't these mutations not updates? I would go for strictly GQL language vs SQL language.

Reply:
sure